inc/class-statify-dashboard.php

```diff
@@ -129,22 +125,25 @@ public static function add_js() {
 		);
 		wp_register_script(
 			'statify_chart_js',
-			plugins_url(
-				'js/dashboard.min.js',
-				STATIFY_FILE
-			),
-			array( 'jquery', 'chartist_tooltip_js' ),
+			plugins_url( 'js/dashboard.min.js', STATIFY_FILE ),
+			array( 'wp-api-fetch', 'chartist_tooltip_js' ),
 			self::$_plugin_version,
 			true
 		);
 
 		// Localize strings.
 		wp_localize_script(
 			'statify_chart_js',
-			'statify_translations',
+			'statifyDashboard',
 			array(
-				'pageview'  => strip_tags( esc_html__( 'Pageview', 'statify' ) ),
-				'pageviews' => strip_tags( esc_html__( 'Pageviews', 'statify' ) ),
+				'i18n'  => array(
+					'error'        => strip_tags( esc_html__( 'Error loading data.', 'statify' ) ),
+					'nodata'       => strip_tags( esc_html__( 'No data available.', 'statify' ) ),
```

Comment:
Doesn’t `strip_tags` and `esc_html` do the same here?

Reply:
No, doesn’t, but after the HTML was escaped, the `strip_tags` shouldn’t find something

Reply:
Absolutely right. We should remove the `strip_tags` and stick with the commonly used `esc_html` here.

Just moved the block to the `i18n` subarray, but if we modify this code anyway, seems like a good point to clean this up.